test/test_utils.jl

```diff
@@ -2,14 +2,8 @@
 # format is the same as the last version of Julia we've tested against.
 const SERIALIZED_HEADER = if VERSION >= v"1.5"
```

Comment:
```suggestion
const SERIALIZED_HEADER = if VERSION >= v"1.6"
```
The linked MR changed min supported version to `1.6`

Reply:
I did noticed this and kept it as 1.5 as the header happens to be the same. I thought since we already know it's 1.5 compatible so I kept it that way. I can definitely change it just let me know what you prefer.

Reply:
Currently this version will never run on <1.6 due to the restriction in Project.toml. Is it worth even having this check anymore?

Reply:
> Currently this version will never run on <1.6 due to the restriction in Project.toml. Is it worth even having this check anymore?

That's a good point I didn't consider. Probably in some future version of Julia this header will change again but as of now it hasn't change as of Julia 1.5 so we can drop the logic until it's needed again. I'll leave an appropriate comment